docs/index.md

```diff
@@ -21,25 +21,49 @@ DDEV by default uses ports 80 and 443 on your system when projects are running.
 
 ## Installation
 
-### Docker Installation
+### Docker or Colima Prerequisite
 
 Docker or an alternative is required before anything will work with DDEV. This is pretty easy on most environments; see the [docker_installation](users/docker_installation.md) page to help sort out the details, especially on Windows and Linux. It is not required to install docker-compose because DDEV uses its own private version.
 
-### Homebrew - macOS/Linux
+### macOS (Homebrew)
 
-For macOS (both amd64 and arm64) and Linux users, we recommend installing and upgrading via [Homebrew](https://brew.sh/) (macOS) or [Homebrew on Linux](https://docs.brew.sh/Homebrew-on-Linux) (Linux): `brew install drud/ddev/ddev`.
+For macOS (both amd64 and arm64) users, we recommend installing and upgrading via [Homebrew](https://brew.sh/): `brew install drud/ddev/ddev`.
 
-As a one-time initialization, run `mkcert -install`. Linux users may have to take additional actions as discussed below in [Linux `mkcert -install` additional instructions](#Linux-mkcert--install-additional-instructions).
+As a one-time initialization, run `mkcert -install`.
 
 Later, to upgrade to a newer version of DDEV-Local, run `brew upgrade ddev`.
 
-To install the latest unreleased DDEV version, `brew unlink ddev && brew install drud/ddev/ddev --HEAD`.
+To install DDEV prereleases, subscribe to the "edge" channel with `brew install drud/ddev-edge/ddev` and to install the latest unreleased DDEV version, `brew unlink ddev && brew install drud/ddev/ddev --HEAD`.
 
-### Installation or Upgrade - Windows (WSL2)
+### Linux (Arch)
 
-**This is the recommended installation method for all Windows users that are on Windows 10 2004 or higher**. If you don't have this version yet, or if you don't want to use WSL2, please follow the legacy instructions for Windows below.
+We maintain a package on [Arch Linux (AUR)](https://aur.archlinux.org/packages/ddev-bin/).
```

Comment:
Might want to mention that package installation on Arch-based systems is preferable to the install script below.  That was my intention for putting it above the install script instructions.

Reply:
```suggestion
We maintain a package on [Arch Linux (AUR)](https://aur.archlinux.org/packages/ddev-bin/).

**NOTE: Package installation on Arch-based systems is preferable to the install script below.**
```